stable/prometheus-adapter/values.yaml

```diff
@@ -51,6 +51,16 @@ rules:
   # Mounts a configMap with pre-generated rules for use. Overrides the
   # default and custom entries
   existing:
+  external: []
+# - seriesQuery: '{__name__=~"^some_metric_count$"}'
+#   resources:
+#     template: <<.Resource>>
+#   name:
+#     matches: ""
+#     as: "my_custom_metric"
+#   metricsQuery: sum(<<.Series>>{<<.LabelMatchers>>}) by (<<.GroupBy>>)
+  # Mounts a configMap with pre-generated rules for use. Overrides the
```

Comment:
This comment looks duplicated. Also comment above should probably mention existing overrides external (or all rules) as well.

Reply:
👍 Thanks for noticing